### PR TITLE
ci: Change xurls url

### PR DESCRIPTION
**Pull request by chavafg**
xurls repo url is not being resolved. Switch to
the github url.

Fixes: #835.

Signed-off-by: Salvador Fuentes <salvador.fuentes@intel.com>

### CODE REVIEW

**Comment by caoruidong** [comment]
LGTM. This is very urgent.

**Comment by jodh-intel** [comment]
Raised https://github.com/kata-containers/ci/issues/74 for the ARM CI failure (which appears unrelated to this PR).

lgtm

File: .ci/static-checks.sh [hunk 1]
```diff
@@ -385,7 +385,7 @@ check_docs()
 	if [ ! "$(command -v $cmd)" ]
 	then
 		info "Installing $cmd utility"
-		go get -u "mvdan.cc/xurls/cmd/$cmd"
+		go get -u "github.com/mvdan/${cmd}"
```
Comment (jodh-intel) [inline .ci/static-checks.sh]: Ah - actually, this needs to be:

```suggestion
		go get -u "github.com/mvdan/xurls/cmd/${cmd}"
```

**Comment by jodh-intel** [comment]
We can't merge this until the xurls URL is corrected (oh the irony... :smile:)

File: .ci/static-checks.sh (hunk 1 above)
Comment (grahamwhaley) [reply .ci/static-checks.sh]: When I saw this last week, I could not make the github URL work. The 'mvdan.cc' site seemed to be temporarily down, and then came back up, and everything started working for me again, hence I didn't file a fix or dig deeper on how to make the github URLs work.

**Comment by jodh-intel** [comment]
... and the reason the CI tests passed here is that the `xurls` command is only used when the PR changes any documentation files.

So although `go get -u "github.com/mvdan/xurls` "works", it doesn't install the `xurls` binary. For that, you need to `go get -u "github.com/mvdan/xurls/cmd/xurls`.

**Comment by jodh-intel** [comment]
In summary, this fix is urgent, but only new PRs *that change documents* will fail the CI stage until this PR lands.

**Comment by jodh-intel** [comment]
... errmm. Correction: all new PRs *will* fail as the existing URL for xurls is now invalid.

**Comment by caoruidong** [comment]
Yes. All PRs are blocked now.

**Comment by jodh-intel** [comment]
I'll raise a PR that replaces this one so we can resolve this "now"...

**Comment by jodh-intel** [comment]
Closing as replaced by #839.

**Pull request closed without merging**